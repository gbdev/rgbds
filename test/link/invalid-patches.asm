section "zero", rom0
Zero::
db 1 % Zero
db 2 ** (Zero - 1)
db BANK(NonexistentSymbol)
db BANK("NonexistentSection")
db STARTOF("NonexistentSection")
db SIZEOF("NonexistentSection")
ldh [Zero], a
jr Zero + 200
