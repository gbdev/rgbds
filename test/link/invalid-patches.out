error: invalid-patches.asm(10): JR target must be between -128 and 127 bytes away, not 190; use JP instead
warning: invalid-patches.asm(9): LDH is deprecated with values from $00 to $FF; use $FF00 to $FFFF
error: invalid-patches.asm(8): Requested SIZEOF() of section "NonexistentSection", which was not found
error: invalid-patches.asm(7): Requested STARTOF() of section "NonexistentSection", which was not found
error: invalid-patches.asm(6): Requested BANK() of section "NonexistentSection", which was not found
error: invalid-patches.asm(5): Requested BANK() of symbol "NonexistentSymbol", which was not found
error: invalid-patches.asm(4): Exponent by negative value -1
error: invalid-patches.asm(3): Modulo by 0
Linking failed with 7 errors
